bitnami/parse/README.md

```diff
@@ -343,6 +430,26 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 18.0.0
+
+This major release renames several values in this chart and adds missing features, in order to be inline with the rest of assets in the Bitnami charts repository.
+
+Affected values:
+
+- `server.port` is renamed to `server.containerPorts.http`
+- `service.port` is renamed to `service.dashboard.ports.http`
+- `metrics.containerPort` is renamed to `metrics.containerPorts.metrics`
+- `service.nodePort` is renamed to `service.dashboard.nodePorts.http`
```

Comment:
Here you mention `service.dashboard.*`, but the value is `dashboard.service` if I'm not mistaken.